ui/spinner.js

```diff
@@ -258,17 +274,13 @@ return $.widget( "ui.spinner", {
 		return false;
 	},
 
-	_uiSpinnerHtml: function() {
```

Comment:
This is documented as an extension point: http://api.jqueryui.com/spinner/#method-_uiSpinnerHtml